config/application.rb

```diff
@@ -57,9 +57,7 @@
 module Mastodon
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 7.0
-
-    config.active_record.marshalling_format_version = 7.1
```

Comment:
This was previously set here to work around a config bug where it didn't get set correctly from the new_framework_defaults file. With bump to 7.1 defaults, it isn't needed.